### ISSUE TITLE
AirParif's air quality data not displayed

### ISSUE THREAD

**Issue by CelianGdfrd** [WangDaYeeeeee/GeometricWeather#456]
Even though the card is set to show, GeometricWeather doesn't display an air quality card, provided by AirParif.

**Comment by papjul** [comment]
Only Auvergne-Rhône-Alpes is supported. See also here comparison of features: https://github.com/papjul/GeometricWeather/blob/new/README.md

Contributions welcomed :)

**Comment by CelianGdfrd** [comment]
Well, AirParif appears in the settings, with the possibility to setup a custom API key.

**Comment by papjul** [comment]
Yes, but this key is not used anywhere. I guess we could hide the ability to change the key until it is implemented.

**Comment by CelianGdfrd** [comment]
Oh, I guess that solves it then.

Thanks a lot !
